samples/MvcSandbox/.gitignore

```diff
@@ -1,2 +0,0 @@
-# Ignore everything
```

Comment:
These changes weren't supposed to be part of this PR. I will revert. I was trying these because this gitignore content doesn't work for me at all. i.e i was expecting any changes to files in MvcSandbox would be ignored by Git, but they are not. Does this work for you?(reading SO posts suggests doing some changes to one's own git index). @ryanbrandenburg , any ideas here?